release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,14 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 9 December 2024 - Enhancement - Home/Admin/Catalog/Marketplace/Sharing - Native link element for app titles [ID 51524]
+
+The app title of the following apps: Admin, Catalog, Home, Marketplace & Sharing now use native link element, allowing users to open the apps in a new tab using middle-click or right-click options.
```

Comment:
Could you rephrase this so it's more from the perspective of the user? "Native link element" might not mean much to them. Note also that the ID for this release note doesn't seem to exist, so possible there's a mistake there.

Reply:
Oh, my bad, first number should be a 4